pip/qsharp/_qsharp.py

```diff
@@ -317,6 +322,9 @@ def on_save_events(output: Output) -> None:
         # compilation.
         entry_expr = None
 
+    durationMs = (monotonic() - start_time) * 1000
```

Comment:
Do you want to clamp this to integral ms or leave it as float?

Reply:
I though float was fine. Any reason to clamp it?

Reply:
Do we care about fractional ms for querying? Unless we need float, I would normally use int for faster processing.

Reply:
This will happen once after other expensive processing (e.g. a simulation or resource estimation). I don't think the difference between an int or a float for this expression would be observable.